apostrophecms/apostrophe
# 3/join browse button

# Browse button

## How to test

- in A3 demo, `npm run dev`
- have existing products, create or edit an article
- go to the `ungrouped` tab in the article, and click on "Browse"
- add, remove, reorder joined products
- check if `max` limit is respected
- check if existing joins are updated and saved correctly
